### DISCUSSION TITLE
is it possible to create Lake Effect Snow?

### DISCUSSION THREAD

**Discussion by MET905** [niels747/2D-Weather-Sandbox#27] [Q&A] [answered]
Whats the best way to create lake effect snow?

**Comment by Meso22** [comment]
You need cold air below zero and warm water but not too warm try to stay below 15 degrees.

**Comment by JustLearninglol** [comment, marked as answer]
Get a cold front below freezing to move over a large body of water (the lake) and make sure the water is warmer than the air coming/surrounding it. This would lead to lake effect snow or rain. The warmer the body of water is, the greater and heavier the lake effect snow would be. The colder the body of water is, less lake effect snow.

**Reply by niels747** [reply]
Indeed, and it also helps to turn up the Lake / Sea Evaporation.

**Reply by MET905** [reply]
Thank you! I just created a massive snow squall over the lake! My issue was I had a capping inversion stopping convection a few cold fronts fixed the problem. Niels Thank you for building this amazing Sim.

**Reply by JustLearninglol** [reply]
I have a lot of fun making cold fronts go over lakes and seas because it generates rainstorms and snowstorms that can last a while. Make a cold front vigorous and it can really crank up that lake effect machine.

**Reply by MET905** [reply]
Me to! the stronger the temperature gradient, the more intense convection gets! I have a snow squall that's hammering a town right now, This is very realistic because just like real life, Upwind of the lake has the most intense precipitation. rain snow mix by the shore but inland its snowing very hard. the band is Starting to weaken now because the winds are easing. no more fetch off the lake.

**Reply by MET905** [reply]
I just have one more question about this topic. Im trying to use the wind tool and it does not seem to be working. I am dragging it around and nothing is happening. im trying to make my intense snow band drift to the other side of the map.

**Reply by JustLearninglol** [reply]
that will kill the band. also u drag the wind tool i think

**Reply by MET905** [reply]
I was using the wind tool and nothing happens lol! My squall finally did die though after warm air took over lol! 
I had no idea it was possible to create such a unique meteorological Phenomenon! my map is based off Niagara Falls (My hometown) with Lake Erie and Ontario.

**Reply by niels747** [reply]
The wind tool is not very responsive indeed, but make sure to set the brush intensity to the max. 
I really love that you guys are having so much fun with my simulator. I haven't had much time to work on it lately, but I am still making small improvements behind the scenes. I will never stop working on this, it's my passion!

**Reply by JustLearninglol** [reply]
Yes your simulation is incredible. The wind tool is very useful in pushing humidity from the ocean or sea inland to generate heavy thunderstorms. Can't wait for the next update. Make it so fires are contained by rainstorms, those things burn literally everything when you leave them alone.

**Reply by niels747** [reply]
Fires need some work indeed, now they even burn the sea 😅

**Reply by JustLearninglol** [reply]
ill make sure not to put vegetation near the coast when I start fires :P

**Reply by MET905** [reply]
Thank you for being so Responsive Niels! I have created pretty much all weather. I think its amazing how realistic everything is, I have looked for something like this for years! I'm a first responder but I have always loved weather since I was a kid, I have created little sims myself but nothing like this!! Meteorology rocks! Can't wait for lightning to be added!!

**Comment by MET905** [comment]
the beginning of the monster snow squall I created over Lake Erie. 
Deep convection and perfect low / mid level winds are keeping the convection organized. 

![snowsquall](https://user-images.githubusercontent.com/113155608/203437201-b0f2549d-5339-4644-899c-1401c3d65a9b.jpg)

**Reply by Meso22** [reply]
Do you have a save file for this?

**Comment by MET905** [comment]
After creating a nasty Arctic air blast the map is primed for Lake effect snow. I also see a meso low starting over the lake. 
its about to get real nasty for people at the lee of Lake Erie lol 

Enjoy ;) 
https://www.dropbox.com/s/jrhyqx110visrkv/Lake%20Erie%20Frontal%20Snow%20Squall%20Event.weathersandbox?dl=0

**Reply by Meso22** [reply]
Thanks Dude!

**Reply by MET905** [reply]
NP, Increase the Lake temp and send another shot of cold air over the lake and watch the monster Snow squall begin. the lake effect band has gotten so massive that it covers the entire map now lol!

**Reply by MET905** [reply]
be easy with that cold front though, just a little cold air is all you need. If you go to crazy your going to create high wind shear and pretty much destroy the band. LOL

**Reply by Meso22** [reply]
Sometimes it is very difficult to create large storms so I just let mine form naturally and it took 18 days in real life.

**Reply by JustLearninglol** [reply]
Thank you for the great lakes! TIME TO MESS AROUND!

**Reply by MET905** [reply]
hmmm it should not take that long. Do you have a capping inversion in place?

**Reply by JustLearninglol** [reply]
the map is huge. very low. fps for me lol

**Reply by MET905** [reply]
Sorry about that I have a beefy PC. I can make you a smaller map if you like.

**Reply by JustLearninglol** [reply]
yes please. that would be lovely.

**Reply by niels747** [reply]
Really nice! Unfortunately I only have my slow laptop with me right now so I can't run it fast.

**Reply by MET905** [reply]
Sorry guys I was at work, Tomorrow I'm going to build the map for those with slower PC's

**Comment by MET905** [comment]
I'm going to build a map that has the north pole and the golf of Mexico so people can mess with air masses and create some nasty storm systems.

**Reply by niels747** [reply]
Sounds like you need more than 6000 horizontal resolution... 
That is just an arbitrary limit I set because it would run to slow on any hardware with higher values.
The next version will have adjustable vertical resolution again, from 100 to 300.  I think I will increase the horizontal limit to 10000.
A map of 100 X 10000 would be 12 km X 1200km, some serious distance!
The solar angle will however have to be the same along the entire plane, because the map is flat, it's very hard to make it curved like the earth.

**Reply by JustLearninglol** [reply]
make the horizontal distance 1 million, so when we want to get rid of old computers, we just run your simulation on them and boom! They are good for the garbage.

**Reply by MET905** [reply]
Hey Niels, what cases a storm system to lose moisture in your sim? i have been trying to create a basic storm system with heavy rain but the clouds keep evaporating lol

**Reply by niels747** [reply]
Have you turned global drying to 0.0 it removes water from the air. The only other water leaves the air is as precipitation.

**Comment by MET905** [comment]
Hey Fellow Weather enthusiasts! As promised here is the Lake Erie Weather Project 

The great lakes have an incredible role in how our weather is here in Ontario Canada, In the summer we have lake breeze convection and in the winter we have Lake Effect Snow squalls that develop to the lee of the lakes. 

first... If you want to start a lake breeze convergence Thunderstorm. heat the ground so its nice and hot but make sure the lakes are colder than land and wait for the winds to pick up due to the heating. you will notice cumulus clouds start to develop at the shore and mature into weak / Moderate thunderstorms. SOME storms could become severe but the right ingredients have to be in place. 
moving on...

Snow Squalls!!!

for this to happen the air around the lake MUST Be colder than the lake. second either create a cold front in the freezing temps OR let the winds naturally develop over the lake, Its important that you make the lake warmer than the air around it! if this does not happen lake effect snow will not happen. I suggest if you want stronger bands to develop keep the winds relatively light and try to avoid high shear, that will destroy the snow bands and they will end up just being local bursts of snow. the linear type bands are more intense and will be able to keep going creating major events! I also added a rise of land at the shore that will help with deep orthographic lift ENJOY AND HAVE FUN! 

https://www.dropbox.com/s/vlot7malmi1giev/Lake%20Erie%20Weather%20Project.weathersandbox?dl=0
